src/OnlineRetailPortal.Core/Models/GetProductCoreResponce.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace OnlineRetailPortal.Core
+{
+    public class GetProductsCoreResponce
```

Comment:
We do not need response in core model,
We can create getproductresponse at service level using core models 
You don’t need explicit core response class

Reply:
Yes but we are giving paging info in response so GetProductCoreResponse consist of List<Product> PagingInfo properties